### PR TITLE
Show retests avoided

### PR DESCRIPTION
**Pull request by eparis**
And some other minor display updates

### CODE REVIEW

**Comment by eparis** [comment]
@lavalamp

File: mungegithub/submit-queue/www/script.js [hunk 1]
```diff
@@ -61,25 +61,43 @@ function SQCntl(dataService, $interval, $location) {
...
+    for (var i = 0; i < reloadFuncs.length; i++) {
+      var func = reloadFuncs[i];
+      if (self.functionLoading[func] == true) {
+        continue;
       }
-      self.tabLoaded[self.selected] = true;
+      self.functionLoading[func] = true;
+
+      var updateIntervalMinutes = reloadFunctions[func];
+      func();
```
Comment (lavalamp) [inline mungegithub/submit-queue/www/script.js]: I don't really know js... is this really a thing??

Comment (lavalamp) [inline mungegithub/submit-queue/www/script.js]: Oh, I see. It's calling func. Github's syntax highlighting confused me.

**Comment by lavalamp** [comment]
I have no idea if this is reasonable js or not but I don't see anything obviously wrong. LGTM, merge if you're confident.

File: mungegithub/submit-queue/www/script.js (hunk 1 above)
Comment (eparis) [inline mungegithub/submit-queue/www/script.js]: yeah, this calls `func` 'right now' and then the next line calls it 'every so often'

**Pull request merged**